recipes/colormap/build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
Please consider removing the `build.sh` file and using the `script` option in the `meta.yaml` file as is done here:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L26